aiida/engine/processes/builder.py

```diff
@@ -7,10 +7,11 @@
 # For further information on the license, see the LICENSE.txt file        #
 # For further information please visit http://www.aiida.net               #
 ###########################################################################
-# pylint: disable=cell-var-from-loop
+# pylint: disable=cell-var-from-loop, line-too-long
 """Convenience classes to help building the input dictionaries for Processes."""
 import collections
 
+from aiida.orm import Node
```

Comment:
Why are you importing this?

Reply:
It was being imported below (see [here](https://github.com/aiidateam/aiida-core/pull/4419/files#diff-72e781fa100001865d9fa09b91d92567L163)) but pylint was complaining and wanted me to move the import to the top of the file.